lib/java/com/google/android/material/datepicker/MaterialDatePicker.java

```diff
@@ -240,6 +260,11 @@ public final View onCreateView(
       confirmButton.setEnabled(false);
     }
     confirmButton.setTag(CONFIRM_BUTTON_TAG);
+    if (positiveButtonTextResId != 0){
```

Comment:
Switch the order to respect text first. This is aligned with how title works.

Reply:
Fixed